test/e2e/framework/fixture.go

```diff
@@ -396,7 +396,7 @@ func (sf SyncerFixture) Start(t *testing.T) *StartedSyncerFixture {
 	// Write the upstream logical cluster config to disk for the workspace plugin
 	upstreamRawConfig, err := sf.UpstreamServer.RawConfig()
 	require.NoError(t, err)
-	_, kubeconfigPath := WriteLogicalClusterConfig(t, upstreamRawConfig, sf.WorkspaceClusterName)
+	_, kubeconfigPath := WriteLogicalClusterConfig(t, upstreamRawConfig, sf.WorkspaceClusterName, TestConfig.AdminContext())
```

Comment:
falling over this. Why AdminContext? What is this kubeconfig used for?

Note (not introduced in this PR but preexisting): we should always qualify names. Kubeconfig says nothing. Is it upstream? Downstream? Is it for fake-downstream logical cluster?

Reply:
i was under the impression all e2e tests have to use the same context, apparently i was wrong?